Main.postman_collection.json

```diff
@@ -403,6 +421,107 @@
 						}
 					]
 				},
+				{
+					"name": "Recording Upload URL",
+					"protocolProfileBehavior": {
+						"disabledSystemHeaders": {}
+					},
+					"request": {
+						"method": "POST",
+						"header": [],
+						"body": {
+							"mode": "raw",
+							"raw": "{\n  \"externalId\": \"123\",\n  \"organisationId\": 1,\n  \"startDate\": \"2022-01-01\",\n  \"uploadUseType\": \"Video On Demand\",\n  \"fileDownloadURL\": \"{{fileDownloadURL}}\",\n  \"onFinishedMatchingCallBackURL\": \"{{callbackServerUrl}}\"\n}",
+							"options": {
+								"raw": {
+									"language": "json"
+								}
+							}
+						},
+						"url": {
+							"raw": "{{lvsApi}}/revenue-streams/upload-url",
+							"host": [
+								"{{lvsApi}}"
+							],
+							"path": [
+								"revenue-streams",
+								"upload-url"
+							]
+						},
+						"description": "This endpoint should be used to deliver recordings for verification via a provided URL (fileDownloadURL). All files will be fetched via a simple get requests so if the download requires authentication it will need to be in the URL. The fetch has a 15minute timeout to download and convert the file so it is important that the download speed of the file is good.\n\nThis endpoint accepts any FFMPEG compatible file formats and will download and strip just the audio element before verification.  \nBody Schema:\n\n```\n{ \n//startDate - the date the recording became live \nstartDate: string\n//endDate - (optional) the date the recording will be taken down \nendDate?: string \n//externalId - (optional) the external id of the sound recording \nexternalId?: string \n//organisationId - the ClicknClear id of the organisation who this sound recording belongs to. \norganisationId: number\n//fileDownloadURL - a link to the file to be uploaded \nfileDownloadURL: string \n//uploadUseType - how this sound recording will be used \nuploadUseType: \"Video On Demand\" | \"Live Stream\" \n//onFinishedMatchingCallBackURL - (optional) an optional call back url which will be called via POST request once the upload has finished the matching process. The shape of the data posted is the same as the Get Revenue Stream Recording Request \nonFinishedMatchingCallBackURL?: string \n}\n\n```"
```

Comment:
NIT: "a" within "a simple get requests" implies singular, should this be omitted such that it reads "via simple get requests..."?

Reply:
Resolved by: 2954ce0b2e1c6252e016a41af16d57ce13404434